parsers/FR.py

```diff
@@ -171,7 +171,7 @@ def fetch_price(zone_key, session=None, target_datetime=None,
                 if value.text == 'ND':
                     continue
                 period = int(value.attrib['periode'])
-                datetime = start_date.replace(hours=+period).datetime
+                datetime = start_date.replace(hour=+period).datetime
```

Comment:
```suggestion
                datetime = start_date.shift(hours=+period).datetime
```

see https://github.com/tmrowco/electricitymap-contrib/issues/2416#issuecomment-619422705